Makefile

```diff
@@ -7,6 +7,7 @@ PREV_VERSION=0.3.0
 VERSION=0.4.0
 COMPILE_TARGET=./tmp/_output/bin/$(IMAGE_NAME)
 OPERATOR_SDK_VERSION=0.12.0
+DEFAULT_ORGANIZATION_TAG=integreatly.org/
```

Comment:
i'm thinking we can rename this to something more generic, such as 

```
TAG_KEY_PREFIX
```

if unset or empty, we can default to `integreatly.org/` instead of requiring it to be set in the Makefile or as an env var. if that makes sense?

Reply:
:+1: